starter-code/scripts/articleView.js

```diff
@@ -33,28 +29,26 @@ articleView.populateFilters = function() {
 
 articleView.handleAuthorFilter = function() {
   $('#author-filter').on('change', function() {
-    // REVIEW: Inside this function, "this" is the element that triggered the event handler function we're
-    //         defining. "$(this)" is using jQuery to select that element, so we can chain jQuery methods
-    //         onto it.
     if ($(this).val()) {
-      // TODO: If the select box was changed to an option that has a value, we need to hide all the articles,
-      //       and then show just the ones that match for the author that was selected.
-      //       Use an "attribute selector" to find those articles, and fade them in for the reader.
-
+      $('article').hide();
+      $('article[data-author="'+ $(this).val() +'"]').show();
     } else {
-      // TODO: If the select box was changed to an option that is blank, we should
-      //       show all the articles, except the one article we are using as a template.
-
+      $('article').show();
```

Comment:
WHOOPS! I forgot my :not(.template) ! Anthony's computer crashed right when we were finishing up, and we thought we'd caught all the changes, but didn't click back into the re-show. Should I resubmit the whole thing?